bulk_indexer.go

```diff
@@ -537,6 +551,13 @@ func (b *BulkIndexer) Flush(ctx context.Context) (BulkIndexerResponseStat, error
 		return resp, fmt.Errorf("error decoding bulk response: %w", err)
 	}
 
+	if b.config.IncludeSourceOnError == Unset {
```

Comment:
nit: a comment here would be nice. In 1 year's time we'll forget why we sanitize on Unset but skipping sanitization on True / False. Or add the comment near the config.